src/includes/configuration/before-breadcrumb-hint/python.mdx

```diff
@@ -6,7 +6,16 @@ def before_breadcrumb(crumb, hint):
         crumb['data']['thread'] = hint['log_record'].threadName
     return crumb
 
-sentry_sdk.init(before_breadcrumb=before_breadcrumb)
+sentry_sdk.init(
+    dsn="___PUBLIC_DSN___",
+
+    # Set traces_sample_rate to 1.0 to capture 100%
+    # of transactions for performance monitoring.
+    # We recommend adjusting this value in production,
+    traces_sample_rate=1.0,
+
```

Comment:
i personally would avoid changing codesamples which don't intend to showcase the setup of sentry. this codesample shows off the `before_breadcrumb` option, and nothing else

Reply:
I thought also about that. And from my own experience when I did those kind of thing (I mean going into a documentation or googling, to find a code snipped that I need from time to time, but always forget) 

I always remember the way how I get to the code sample that I need. For example, I know: I go to docs.sentry.io, go to "configuration", "filtering" and then copy the snippet there, because it has everything I need. (because in this example I use before_send in all my projects)

So that was the rational behind changing it everywhere. But maybe that is overkill...